api/v1alpha1/toolchainconfig_types.go

```diff
@@ -32,6 +32,30 @@ type ToolchainConfigSpec struct {
 	// Contains all member operator configurations for all member clusters
 	// +optional
 	Members Members `json:"members,omitempty"`
+
+	// Contains all the configuration that applies to both host and member operators
+	// +optional
+	Global GlobalConfig `json:"global,omitempty"`
+}
+
+// GlobalConfig contains all configuration that applies for both host and member operators
+// +k8s:openapi-gen=true
+type GlobalConfig struct {
+	// +optional
+	PublicViewer PublicViewerConfig `json:"publicViewer,omitempty"`
+}
+
+// PublicViewer used to enable public-viewer support
+// +k8s:openapi-gen=true
+type PublicViewerConfig struct {
+	// +kubebuilder:default:=false
+	Enabled bool `json:"enabled"`
+
+	// +kubebuilder:default:=public-viewer
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=253
```

Comment:
max should be 63 - it's used as a label value of the SpaceBindings 
```suggestion
	// +kubebuilder:validation:MaxLength=63
```

Reply:
resolved in 7225626b4936c8f72f50eaad24bc272bee268344